src/nl/hannahsten/texifyidea/inspections/latex/LatexFileNotFoundInspection.kt

```diff
@@ -47,7 +44,25 @@ open class LatexFileNotFoundInspection : TexifyInspectionBase() {
     override fun inspectFile(file: PsiFile, manager: InspectionManager, isOntheFly: Boolean): MutableList<ProblemDescriptor> {
         val descriptors = descriptorList()
 
-        val commands = file.commandsInFile()
+
+        val allcommands = file.commandsInFileSet() //get all comands of project
+        val commands = file.commandsInFile()  //get commands of this file
```

Comment:
Comments should go above/before the command they are about, and generally start with a capital (they're often regular sentences). For example, this should be:

```suggestion
        // Get commands of this file.
        val commands = file.commandsInFile()
```

Reply:
Thank you I will do that.